test-cases/scylla-operator/longevity-scylla-operator-3h-multitenant.yaml

```diff
@@ -0,0 +1,39 @@
+test_duration: 300
+prepare_write_cmd:  [
+                     ["cassandra-stress write cl=QUORUM n=209715 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=80 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"],
+                     ["cassandra-stress write cl=QUORUM n=209715 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=80 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"]
+                    ]
+
+stress_cmd: [
+             ["cassandra-stress write cl=QUORUM duration=150m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=40 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5",
+             "cassandra-stress read  cl=QUORUM duration=150m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=40 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"
+             ],
+             [ "cassandra-stress write cl=QUORUM duration=150m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=40 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5",
+               "cassandra-stress read  cl=QUORUM duration=150m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=40 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"
+             ]
+            ]
+
+stress_read_cmd: [
+                  ["cassandra-stress read  cl=QUORUM duration=150m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=40 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"],
+                  ["cassandra-stress read  cl=QUORUM duration=150m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=40 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"]
+                 ]
+
+# NOTE: we disable second type of monitoring - K8S based one. It is redundant while we use standalone ones.
+k8s_deploy_monitoring: false
+
+n_db_nodes: 4
+n_monitor_nodes: 1
+
+# NOTE: we deploy here 4 K8S nodes of the 'loader' type and going to create 2 pairs of loader pods.
+n_loaders: 4
+k8s_n_loader_pods_per_cluster: 2
+
+
+nemesis_class_name: 'SisyphusMonkey'
+nemesis_interval: 5
+
+user_prefix: 'longevity-scylla-operator-3h-multitenant'
+
+space_node_threshold: 64424
+
+cluster_health_check: false
```

Comment:
why we disable the health check ?

Reply:
I suppose that it may be the problem with parallel threads like we got with nemeses. So I disabled it for this time, do not to do things more complicated

Reply:
It sounds like a required part of the feature. Cluster health checks must be ported to be multi-tenant too. IMHO

Reply:
Yes, it have to work. We splitted this to a few and I suggest to do CHC as separate task

Reply:
o.k. please write it down in trello if you didn't.